user_sync/rules.py

```diff
@@ -75,11 +75,26 @@ def __init__(self, caller_options):
             'number_of_users_created': 0,
             'number_of_users_updated': 0,
             'number_of_users_with_updated_groups': 0,
-            'number_of_users_removed_from_mapped_group': 0,
+            'number_of_users_removed_from_mapped_groups': 0,
             'number_of_users_removed': 0,
             'number_of_users_deleted': 0,
             'number_of_users_with_no_changes': 0,
         }
+        # English text description for action summary log.
+        # The action summary will be shown the same order as they are defined in the list
+        self.action_summary_descriptions = [
+            ['total_number_of_adobe_users', 'Total number of Adobe users'],
+            ['number_of_adobe_users_excluded', 'Number of Adobe users excluded'],
+            ['total_number_of_directory_users', 'Total number of directory users'],
+            ['number_of_directory_users_selected', 'Number of directory users selected'],
+            ['number_of_users_created', 'Number of users created'],
```

Comment:
let's be clear in all these "change" statistics that they are "Adobe users", not just "users"